app/src/main/java/com/android/vending/billing/utils/Inventory.java

```diff
@@ -70,9 +70,7 @@ public void erasePurchase(String sku) {
     /** Returns a list of all owned product IDs of a given type */
```

Comment:
This is a third party library, prefer not to make changes here, as it makes it difficult to merge in future changes to this library.